README.md

```diff
@@ -52,6 +52,8 @@ To use the API, you need to get your API key in Grist from Profile Settings. Run
 with `GRIST_API_KEY=<key>` in the shell environment. The key may also be stored to
 `~/.grist-api-key` file.
 
+To access public documents, you may provide an empty string for the API key.
```

Comment:
Is it ok to just not set it?

Reply:
Yes. Rephrased this, and the above paragraph, to clarify.